maven_plugin/src/main/java/io/github/chains_project/maven_lockfile/graph/DependencyGraph.java

```diff
@@ -81,7 +82,9 @@ private static Optional<DependencyNode> createDependencyNode(
         var classifier = Classifier.of(node.getArtifact().getClassifier());
         var checksum = isRoot ? "" : calc.calculateArtifactChecksum(node.getArtifact());
         var scope = MavenScope.fromString(node.getArtifact().getScope());
-        var resolved = isRoot ? ResolvedUrl.Unresolved() : calc.getArtifactResolvedField(node.getArtifact());
+        var repositoryInformation = isRoot
+                ? new RepositoryInformation(ResolvedUrl.Unresolved(), RepositoryId.of(""))
```

Comment:
Data consistency issue - worth creating a static method on `RepositoryInformation` for the "unresolved" use case?